<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UsersDialog</class>
 <widget class="QDialog" name="UsersDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>282</width>
    <height>132</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Bodeasy - Démarrage</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Utilisateurs</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QComboBox" name="cmbUsers">
        <property name="maximumSize">
         <size>
          <width>125</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="btnDelete">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
        <property name="icon">
         <iconset resource="ressources.qrc">
          <normaloff>:/images/delete-icon.png</normaloff>:/images/delete-icon.png</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="txtUser">
        <property name="maximumSize">
         <size>
          <width>125</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="maxLength">
         <number>24</number>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="btnAdd">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Ajouter</string>
        </property>
        <property name="icon">
         <iconset resource="ressources.qrc">
          <normaloff>:/images/add-icon.png</normaloff>:/images/add-icon.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="spacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnStart">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="text">
        <string>Démarrer</string>
       </property>
       <property name="icon">
        <iconset resource="ressources.qrc">
         <normaloff>:/images/icon-start.png</normaloff>:/images/icon-start.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>12</width>
         <height>12</height>
        </size>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>cmbUsers</tabstop>
  <tabstop>txtUser</tabstop>
  <tabstop>btnAdd</tabstop>
  <tabstop>btnStart</tabstop>
  <tabstop>btnDelete</tabstop>
 </tabstops>
 <resources>
  <include location="ressources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>txtUser</sender>
   <signal>returnPressed()</signal>
   <receiver>btnAdd</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>55</x>
     <y>73</y>
    </hint>
    <hint type="destinationlabel">
     <x>201</x>
     <y>70</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnStart</sender>
   <signal>clicked()</signal>
   <receiver>UsersDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>237</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel">
     <x>235</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
